specification/attributes/datetime_requirements.md

```diff
@@ -0,0 +1,34 @@
+# Datetime Requirements
+
+Billing data columns that provide date and time information (datetime related columns) conforming to specified rules and formatting requirements ensure clarity, accuracy and ease of interpretation for both humans and systems.
+
+All datetime related columns, defined in the FOCUS specification, MUST be represented in UTC and follow the ISO 8601 aligned formatting requirements listed below. Provider generated datetime related columns SHOULD adopt the same format requirements over time.
```

Comment:
"datetime" is technically not an English word as far as I know. I believe it's just tech jargon.
No need to say "over time". 
```suggestion
All date/time related columns, defined in the FOCUS specification, MUST be represented in UTC and follow the ISO 8601 aligned formatting requirements listed below. Provider generated date/time related columns SHOULD adopt the same format requirements.
```

Reply:
suggested changes applied